app/models/metric/statistic.rb

```diff
@@ -4,7 +4,14 @@ def self.calculate_stat_columns(obj, timestamp)
     return {} unless obj.respond_to?(:container_images)
```

Comment:
Do we also want to separate these two conditions? (ie. we might have just `:container_groups` or just `:container_images`)